rebasehelper/tags.py

```diff
@@ -25,22 +25,25 @@
 import collections.abc
 import fnmatch
 import re
-from typing import Iterator, Optional, Tuple, cast
+from typing import Dict, Iterator, List, Optional, Tuple, cast
 
 from rebasehelper.spec_content import SpecContent
 from rebasehelper.helpers.macro_helper import MacroHelper
 
 
 class Tag:
-    def __init__(self, section: str, line: int, name: str, value_span: Tuple[int, int], valid: bool) -> None:
-        self.section: str = section
+    def __init__(self, section: str, line: int, name: str, value_span: Tuple[int, int], valid: bool,
+                 section_index: int) -> None:
```

Comment:
Constructor arguments don't match the attributes now.

Reply:
Should I move `section_index` to the first position as well?

Reply:
Yes, of course. Why should it be last?